### PR TITLE
can toggle time alerts on and off

### PR DESCRIPTION
**Pull request by gabrielsaruhashi**
[sc-2903]

image
image

### CODE REVIEW

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx [hunk 1]
```diff
@@ -147,10 +149,33 @@ const VoiceCallInterface = ({ vc, authInfo, onTerminate }: Props) => {
...
+          <IconButton
+            icon={<ClockCircleOutlined className="text-lg" />}
+            type={hasTimeAlert ? "primary" : "default"}
+            ghost={hasTimeAlert}
+            shape="circle"
+            onClick={() => setHasTimeAlert((timeAlert) => !timeAlert)}
+            label={`${t("call:videoOverlay.timeAlert.label")}`}
+            className={
+              hasTimeAlert
+                ? "text-blue-400 border-blue-400"
```
Comment (gabrielsaruhashi) [inline src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:173]: this is needed to override the default "press" state color that antd offers

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx [hunk 2]
```diff
@@ -147,10 +149,33 @@ const VoiceCallInterface = ({ vc, authInfo, onTerminate }: Props) => {
...
+              : t("call:videoOverlay.timeAlert.tooltipAlertOff")
+          }
+        >
+          <IconButton
+            icon={<ClockCircleOutlined className="text-lg" />}
+            type={hasTimeAlert ? "primary" : "default"}
+            ghost={hasTimeAlert}
+            shape="circle"
+            onClick={() => setHasTimeAlert((timeAlert) => !timeAlert)}
+            label={`${t("call:videoOverlay.timeAlert.label")}`}
```
Comment (cainlevy) [inline src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:170]: 🏌️‍♂️ 

```suggestion
            label={t("call:videoOverlay.timeAlert.label")}
```

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx (hunk 1 above)
Comment (cainlevy) [reply src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:173]: should we bake this color choice into the common component? or are there other places in the app that aren't ready for an update.

**Review by cainlevy — APPROVED** [review]

File: src/components/Inbox/VoiceCall/VoiceCallInterface.tsx (hunk 1 above)
Comment (gabrielsaruhashi) [reply src/components/Inbox/VoiceCall/VoiceCallInterface.tsx:173]: this behavior is pretty specific to this implementation bc this button is acting like a toggle :/ 
but yes, we should move to the common component now that liz and jordan are wrapping up the color updates

**Pull request merged**